CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
 
 ### Changed
 
+- Refactored client shop logic into separate shop-class (in Progress)
```

Comment:
```suggestion
- Refactored client shop logic into separate shop-class
```

i get what you want to say with this but i think it's just fine without the 'in progress' label